assessments/0008-backstage.md

```diff
@@ -0,0 +1,403 @@
+
+There is **example content** available, including tutorials for a variety of tasks and a very nice [demo server](https://demo.backstage.io/catalog?filters%5Bkind%5D=component&filters%5Buser%5D=owned).
+
+
+Examples:
+
+* https://falco.org/docs/getting-started/
+* https://prometheus.io/docs/introduction/first_steps/
+
+
+#### Content maintainability & site mechanics
+
+The documentation is **searchable**. There does not seem to be a **localization** framework. There doesn't currently seem to be any kind of localization effort.
+
+There does not seem to be any mechanism for **versioning** documentation content.
+
+Examples:
+
+* https://kubernetes.io/docs/contribute/
+
+#### Content creation processes
+
+The procedures for duplicating the documentation locally and for contributing documentation are [documented](https://github.com/backstage/backstage/tree/master/microsite) but are [not easily findable](https://backstage.io/docs/getting-started/getting-involved#write-documentation-or-improve-the-website).
+
+These procedures are presumably included in any updates to (and subsequent **release** of) the project code (since the doc is in the same GitHub repo). **Reviewers and approvers** are presumably the Backstage OSS community at large. It would be nice if this situation were made clear explicitly.
+
+The website does not have a clear individual **owner or maintainer**.
+
+Examples:
+
+* https://github.com/nats-io/nats-site/blob/master/MAINTAINERS.md (clearly documented maintainers)
+* https://thanos.io/tip/contributing/how-to-contribute-to-docs.md/
+
+#### Inclusive language
+
+I found no content that uses non-recommended words as documented by the [Inclusive Naming Initiative](https://inclusivenaming.org) website.
+
+The website makes occasional use of words like "simple" and "easy", but avoids explicitly ableist language.
+
+
+### Recommendations
+
+The following sections contain recommendations for improvements to the Backstage project documentation. These recommendations are for broad improvements that would greatly increase documentation effectiveness. Further collaboration between technical writers and project leadership is needed to analyze the recommendations into actionable, estimable work efforts.
+
+
+#### Clarify user roles and objectives
+
+Any systemic improvement to the Backstage documentation hinges on clearly defining and documenting user roles. This is a first step in defining any documentation set, but nature of Backstage makes it especially important here.
```

Comment:
I agree with this recommendation. I like how Judy elicited feedback from the in-toto community to help figure out the user roles, is that something that you think would be a useful endevour here too?

Reply:
As we've discussed since, I think it's a vital step that should be added to the assessment framework (or technically, *expanded* in the assessment framework, since it's already a line item in the website maturity requirements).